examples/trianglell/main.rs

```diff
@@ -83,8 +84,8 @@ fn main() {    env_logger::init().unwrap();
         pixel_shader: Some("ps_main"),
     };
 
-    let pipeline_signature = device.create_pipeline_signature();
-    let render_pass = device.create_renderpass();
+    let pipeline_signature = factory.create_pipeline_signature();
```

Comment:
Yes, as it will follow more the vulkan pipeline layout model than root signatures.